_raml/apps/get_shopid.schema.json

```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "http://json-schema.org/draft-03/schema",
+  "title": "Get ShopId Schema",
+  "type": "object",
+  "properties": {
+    "": {
```

Comment:
We don't have a property name here, see also e.g. Get Orders OrderId Schema. Same thing there.